src/utilities/util_dump.c

```diff
@@ -1014,7 +1015,7 @@ dump_explore(WT_CURSOR *cursor, const char *uri, bool reverse, bool pretty, bool
         while (current_arg != NULL) {
             if ((args[i] = malloc(ARG_BUF_SIZE)) == NULL)
                 WT_ERR(util_err(session, errno, NULL));
-            strcpy(args[i++], current_arg);
+            (void)strncpy(args[i++], current_arg, strlen(current_arg));
```

Comment:
I'm getting confused with between `MAX_ARGS` and `ARG_BUF_SIZE`. 
There is no restriction that `strtok()` to return a string < `ARG_BUF_SIZE`.
Despite the aforementioned confusion, I don't think this follows what I understand to be the intention of `strncpy`: that is to prevent writing on memory beyond the bounds of the first argument `dst`.
So using the code as present as a template:
```suggestion
            (void)strncpy(args[i++], current_arg, ARG_BUF_SIZE - 1);
```
Which enforces the bounds of `dst`.
It can be used to detect truncation by checking if the returned pointer points beyond the end of destination buffer.
```
if (strncpy(args[i], current_arg, ARG_BUF_SIZE - 1) >= args[i] + ARG_BUF_SIZE)
                /* fail with error indicating argument was too long */
            i++;
```

Reply:
Thanks for the suggestion, I have updated the code in f44d24d2b787ad52fd44d5e49968d84b6d1475a0.
However, I did not add your last suggestion, happy to discuss this.

Reply:
As discussed, used `memmove` instead: 438629010d077d2803e720413148289c0820d222